packages/core/upload/server/services/upload.js

```diff
@@ -216,16 +226,12 @@ module.exports = ({ strapi }) => ({
 
     const uploadPromises = [];
 
-    // Upload image
-    uploadPromises.push(getService('provider').upload(fileData));
-
     // Generate & Upload thumbnail and responsive formats
     if (await isResizableImage(fileData)) {
       const thumbnailFile = await generateThumbnail(fileData);
```

Comment:
when generating the thumbnail & responsive sizes are we also running the sanitization ?

Reply:
It only runs once when parsing the original file data, downstream transformations assume the original data is already sanitized.